tools/diffscripts/README.md

```diff
@@ -0,0 +1,208 @@
+# CSharp SDK Diff Automation
+
+## Overview
+During the release process we need to examine the changes to the SDK surface to make sure we're not making too many changes, breaking or otherwise. This tool will automate the process of generating the mark down files so we do not need to use the GUI version which is time consuming.
+
+
+## Quickstart
+This quickstart assumes that you clone your repositories to `c:\repos` and that you've already cloned the CSharp SDK to `c:\repos\azure-iot-sdk-csharp`
```

Comment:
how about we include the api surface within this repo directly? track 2 sdks already do that, and it'll save us from having to clone multiple repos.

Reply:
This would be a change to the process we do now. I'm good with it but not sure what the implications are if we change h to e location of the API docs.